### PR TITLE
Kiera development

### PR DESCRIPTION
**Pull request by kierabr1**
I modified DataServlet.java to return some hard-coded JSON greetings in different languages, and then I modified script.js to fetch that JSON and build a UI from it to display on the home page.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 public class DataServlet extends HttpServlet {
 
   
-  private List<String> quotes;
+  private ArrayList<String> quotes;
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Better to leave this as List. 

You can also make this 
private final List<String> quotes = new ArrayList<String>(). And remove the new from the init method.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -28,8 +28,28 @@ function addRandomFunFact() {
...
+  quoteListElement.appendChild(
+      createQuoteElement('Spanish: ' + quotes.Spanish));
+  console.log(quotes.Spanish);
+  quoteListElement.appendChild(
+      createQuoteElement('French: ' + quotes.French));
+  console.log(quotes.French);
+  quoteListElement.appendChild(
+      createQuoteElement('Hawaiian: ' + quotes.Hawaiian));
+  console.log(quotes.Hawaiian);
   });
```
Comment (sbahaud) [inline portfolio/src/main/webapp/script.js:47]: remove the console.log, if you don;t need them anymore.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -28,8 +28,28 @@ function addRandomFunFact() {
 }
 
 function getRandomQuote() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('quote-container').innerText = quote;
+  fetch('/data').then(response => response.json()).then((quotes) => {
+
+  const quoteListElement = document.getElementById('quote-container');
+  quoteListElement.innerHTML = '';
```
Comment (sbahaud) [inline portfolio/src/main/webapp/script.js:34]: Is the innerHtml = ''" required?

**Review by sbahaud — APPROVED** [review]
Update the PR description. Also run the linter that I shared, to make sure there are no formatting errors.

LGTM

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (kierabr1) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Done

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (kierabr1) [reply portfolio/src/main/webapp/script.js:47]: Done

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (kierabr1) [reply portfolio/src/main/webapp/script.js:34]: Done

**Pull request merged**